### DISCUSSION TITLE
2022-03-16 Weekly Meeting 47 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#199] [Weekly Meetings]
Weekly Meeting 47 is happening on Wednesday March 16th, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates
This will be a 2 hour workshop addressing the OMI structure and leadership.
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by LightLodges** [comment]
Today's meeting starts 30 minutes earlier at 8:30AM and is the first of 2 strategic planning workshops scheduled where we'll be driving into roles and inquiry around the community organization structures we'll want to move toward in the coming year.

**Comment by robertlong** [comment]
### Comment-only Miro Link
https://miro.com/app/board/uXjVOFdPAIQ=/?invite_link_id=103696866511

### Miro Board PDF
[OMI Group Organization and Leadership Workshop.pdf](https://github.com/omigroup/omigroup/files/8266891/OMI.Group.Organization.and.Leadership.Workshop.pdf)

### Miro Backup
[OMI Group Organization and Leadership Workshop.rtb.zip](https://github.com/omigroup/omigroup/files/8266896/OMI.Group.Organization.and.Leadership.Workshop.rtb.zip)
